File: Data Mine Fall Seminar/Project 1/anthony-micco-project01.py
import socket
print(socket.gethostname())

import pandas as pd
df = pd.read_csv('/anvil/projects/tdm/data/flights/subset/1991.csv')
df[df["Month"]==12].head() # see information about a few of the flights from December 1991

#Calculate memory in bytes in Anvil sub-cluster A (Python)
256000*1000000000

#Calculate memory in TB in Anvil sub-cluster A (Python)
256000/1000

File: Data Mine Fall Seminar/Project 1/anthony-micco-project01.R
system("hostname", intern=TRUE)

#Calculate memory in bytes in Anvil sub-cluster A
256000*1000000000

#Calculate memory in TB in Anvil sub-cluster A
256000/1000

dat <- read.csv("/anvil/projects/tdm/data/disney/flight_of_passage.csv")
head(dat)
flight_of_passage <- dat
head(flight_of_passage)
rm(dat)
head(dat)